authorize-idp-access-tokens/README.md

```diff
@@ -0,0 +1,96 @@
+# Authorize IdP Access Token
+
+This sample allows you to authorize JWT access tokens issued by an OIDC complient identiy provider.
+
+## About JWT Access Tokens
+
+JWT tokens are compact and self-contained way to represent and transmit clains between two parties based on the [RFC7519](https://www.rfc-editor.org/rfc/rfc7519) standard. These claims are encoded in JSON format and this data can be digitally sigeded, encrypted, and or integrity protected.
+
+JWT tokens are used by [OIDC](https://openid.net/specs/openid-connect-core-1_0.html) compliant identity providers for:
+* Authentication: After an End-User has been successfully authenticated an authorization server will issue a JWT Id Token to store End-User claims.
```

Comment:
Authentication: After an End-User has been successfully authenticated, an authorization server will issue a JWT Id Token to store End-User claims.

added a "comma"